cfpb/consumerfinance.gov
# Ask CFPB: Update search results count to 25 per page

See https://[GHE]/CFPB/el-camino/issues/365

## Changes

- Ask CFPB: Update portal search results count to 25


## How to test this PR

1. Search on http://localhost:8000/consumer-tools/auto-loans/answers/basics/ should have 25 results or less per page. Compare to 10 results or less on production https://www.consumerfinance.gov/consumer-tools/auto-loans/answers/basics/
2. Search on http://localhost:8000/ask-cfpb/search/?q=money-as-you-grow should have 25 results or less per page. Compare to 20 results or less on production https://www.consumerfinance.gov/ask-cfpb/search/?q=money-as-you-grow
2. Search on http://localhost:8000/ask-cfpb/search-by-tag/money-as-you-grow/ should have 25 results or less per page. Compare to 20 results or less on production https://www.consumerfinance.gov/ask-cfpb/search-by-tag/money-as-you-grow/
